tracer/test/test-applications/security/Samples.Security.AspNetCore5/Samples.Security.AspNetCore5.csproj

```diff
@@ -9,6 +9,30 @@
     <DebugSymbols>false</DebugSymbols>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netcoreapp3.0|AnyCPU'">
```

Comment:
it looks like this sets `<DebugType>none` for every framework we support, so could be simplified? Also, if we always use the fully qualified name, why do we need this change.

Reply:
We still use the line number to calculate the hash, and it is present when debug symbols are generated